app/services/preserved_object_handler.rb

```diff
@@ -73,33 +73,32 @@ def check_existence
 
           raise_rollback_if_pc_po_version_mismatch(pres_copy.version, pres_object.current_version)
 
-          ran_moab_validation = false
-          if incoming_version == pres_copy.version
+          incoming_vers_eq_pc_vers = (incoming_version == pres_copy.version)
+          incoming_vers_gt_pc_vers = (incoming_version > pres_copy.version)
+          found_expected_version = (incoming_vers_eq_pc_vers || incoming_vers_gt_pc_vers)
+          check_status_if_not_ok_or_unexpected_version(pres_copy, found_expected_version)
+
+          if incoming_vers_eq_pc_vers
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_copy.class.name)
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_object.class.name)
-          elsif incoming_version > pres_copy.version
+          elsif incoming_vers_gt_pc_vers
             handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_copy.class.name)
             handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_object.class.name)
-            ran_moab_validation = true
+            # we're not too worried if the version increased, but we normally expect it to be kept current
+            # by calls from ingest robots, so validate to make sure all's well.
```

Comment:
not sure this comment is giving us anything -- i would prefer to read the code to figure it out.  ("so if the incoming is greater then it validates ...")